tests/SchemaFactoryTest.php

```diff
@@ -168,9 +169,8 @@ public function testException2(): void
         $factory->createSchema();
     }
 
-    private function doTestSchema(Schema $schema): void
+    private function _doTestSchema(Schema $schema): ExecutionResult
```

Comment:
Let's not name private methods with a leading underscore, this is a really outdated code style that's really no longer used.